src/templates/resources/footer.json

```diff
@@ -2544,50 +2544,50 @@
         "cuisines": "[Ţöþẋ çûîšîñéšẋẋẋ]",
         "cuisineLinks": [
             {
-                "url": "/chinese-takeaways",
+                "url": "/takeaway/nearme/chinese",
                 "text": "[Çĥîñéšéẋẋẋ]"
             },
             {
-                "url": "/indian-takeaways",
+                "url": "/takeaway/nearme/indian",
                 "text": "[Îñðîåñẋẋ]"
             },
             {
-                "url": "/italian-takeaways",
+                "url": "/takeaway/nearme/italian",
                 "text": "[Îţåļîåñẋẋẋ]"
             },
             {
-                "url": "/sushi-takeaways",
-                "text": "[Ĵåþåñéšéẋẋẋ]"
+                "url": "/takeaway/nearme/sushi",
+                "text": "[Šûšĥîẋẋ]"
             },
             {
-                "url": "/pizza-takeaways",
-                "text": "[Þîžžåẋẋ ðéļîṽéŕýẋẋẋ]"
+                "url": "/takeaway/nearme/pizza",
+                "text": "[Þîžžåẋẋ]"
             },
             {
-                "url": "/cuisine",
+                "url": "/takeaway/nearme",
                 "text": "[Ṽîéŵẋẋ åļļẋ çûîšîñéšẋẋẋ]"
             }
         ],
         "towns": "[Ļöçåţîöñšẋẋẋ]",
         "locationLinks": [
             {
-                "url": "/birmingham-takeaway",
+                "url": "/takeaway/birmingham",
                 "text": "[Ɓîŕɱîñĝĥåɱẋẋẋẋ]"
```

Comment:
Are these mad characters a github thing?

Reply:
It's a [pseudo locale](https://docs.microsoft.com/en-us/windows/desktop/intl/pseudo-locales)